src/Shell/Task/ModelTask.php

```diff
@@ -175,6 +175,11 @@ public function all()
      */
     public function getTableObject($className, $table)
     {
+        $plugin = $this->param('plugin');
+        if (!empty($plugin)) {
+            $className = $plugin . '.' . $className;
```

Comment:
If you tell me how? All I know is that testing console stuff is pain in the ass. :(